libethereum/Block.cpp

```diff
@@ -688,7 +689,7 @@ void Block::performIrregularModifications()
 		Addresses allDAOs = childDaos();
 		for (Address const& dao: allDAOs)
 			m_state.transferBalance(dao, recipient, m_state.balance(dao));
-		m_state.commit();
+		m_state.commit(false);
```

Comment:
Yes, let's convert this into an enum - although I'm not sure if `: bool` is required.